examples/ex0p.cpp

```diff
@@ -47,7 +48,10 @@ int main(int argc, char *argv[])
    H1_FECollection fec(order, mesh.Dimension());
    ParFiniteElementSpace fespace(&mesh, &fec);
    HYPRE_BigInt total_num_dofs = fespace.GlobalTrueVSize();
-   if (mpi.Root()) { cout << "Number of unknowns: " << total_num_dofs << endl; }
+   if (Mpi::Session().Root())
```

Comment:
How about shortening `Mpi::Session().Root()` to `Mpi::Root()`?

Reply:
I am OK with this and the similar suggestions below. @pazner, what do you think?

Reply:
Sure, I can make the change if you want.

Reply:
that will be great, thank you

Reply:
Done, hopefully it's OK now

Reply:
thanks @pazner